packages/patternfly-4/react-docs/src/pages/index.js

```diff
@@ -3,6 +3,7 @@ import Content from '../components/content';
 import { Title } from '@patternfly/react-core';
 import { StyleSheet, css } from '@patternfly/react-styles';
 import packageJson from '../../../react-core/package.json';
+// import '@patternfly/react-core/../dist/styles/base.css';
```

Comment:
This is commented out, should remove if not needed anymore.

Reply:
Was this not getting inserted before? It should be using css loader and should be getting injected the same way that the inject global is doing it

Reply:
No i had added it in some prior testing. base.css is only loaded in the layouts